methods/dropWhile/index.js

```diff
@@ -0,0 +1,29 @@
+import drop from '../drop';
+
+/**
+ * Drops elements up to, but not including, the first element
+ * for which the predicate returns null or false
+ * and returns an array containing the remaining elements.
+ *
+ * If no predicate is given, the array is returned instead.
+ * @param {Array} array
+ * @param {Function} predicate
+ *
+ * @return {Array} array containing the remaining elements
+ *
+ * @example
+ * dropWhile([1, 2, 3, 4, 5, 0], x => x < 3); // => [3, 4, 5, 0]
+ *
+ * @example
+ * rbjs([1, 2, 3, false, 5].dropWhile(x => x < 3);  // => [false, 5]
+*/
+export default function dropWhile(array, predicate) {
+    if (!predicate) {
+        return array;
+    }
+    let index = 0;
```

Comment:
@aldnav Do you think the `index` value can be obtained using [`array.findIndex`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex)?

Reply:
@arnellebalane thanks. Updated.

Reply:
@aldnav Awesome! 🎉